customscriptextension.ps1

```diff
@@ -73,6 +73,9 @@ Expand-Archive -Path ./$zip -DestinationPath .
 Add-LocalGroupMember -Group Administrators -Member "NT AUTHORITY\NetworkService"
 .\config.cmd --unattended --url https://dev.azure.com/$account --auth PAT --token $PAT --pool "$PoolName" --agent "$agentName" --runAsService
 
+Write-Information "###### Docker Service Config ######"
+SC failure docker reset=0 actions=restart/60000/restart/60000/run/60000 command=""shutdown" "/T00""
```

Comment:
```suggestion
SC failure docker reset=0 actions=restart/60000/restart/120000/run/240000 command=""shutdown" "/T00""
```

Reply:
Add this after your changes so if the service is not running it starts it else restart the service?

Reply:
would need to use $Service in the logic, It might be ok to just do a Get-Service $varService | Restart-Service

if the service isnt running it will just start it.